packages/no-case/src/index.ts

```diff
@@ -5,10 +5,12 @@ export interface Options {
   stripRegexp?: RegExp | RegExp[];
   delimiter?: string;
   transform?: (part: string, index: number, parts: string[]) => string;
+  separateNumbers?: boolean;
 }
 
 // Support camel case ("camelCase" -> "camel Case" and "CAMELCase" -> "CAMEL Case").
 const DEFAULT_SPLIT_REGEXP = [/([a-z0-9])([A-Z])/g, /([A-Z])([A-Z][a-z])/g];
+const SEPARATE_NUMBERS_SPLIT_REGEXP = [/([a-z0-9])([A-Z])/g, /([A-Z])([A-Z][a-z])/g, /([(0-9)])([A-Za-z])/g, /([A-Za-z])([0-9])/g,];
```

Comment:
What about appending the two additions you want to the regex list when the option is specified? Also, can we get some more test coverage, specifically around the final regex (number then letter) and combinations of things like `Foo12019Bar` or `V1Test`.

Reply:
```suggestion
const SEPARATE_NUMBERS_SPLIT_REGEXP = [/([a-z0-9])([A-Z])/g, /([A-Z])([A-Z][a-z])/g, /([0-9])([A-Za-z])/g, /([A-Za-z])([0-9])/g,];
```

Typo in the number group.